static/src/javascripts/projects/common/modules/commercial/liveblog-dynamic-adverts.js

```diff
@@ -0,0 +1,101 @@
+define([
+    'bonzo',
+    'common/utils/fastdom-promise',
+    'common/utils/config',
+    'common/utils/mediator',
+    'common/modules/commercial/create-ad-slot',
+    'common/modules/article/space-filler',
+    'common/modules/commercial/dfp-api'
+], function (
+    bonzo,
+    fastdom,
+    config,
+    mediator,
+    createAdSlot,
+    spaceFiller,
+    dfp
+) {
+    var INTERVAL = 5;      // number of posts between ads
```

Comment:
I'm not a fan of magic numbers, that's why. It's worth mentioning that when we start coding in ES6 lingo, this code will go straight to:

```
const INTERVAL = 5;
const OFFSET = 1.5;
```